torchgeo/datasets/cms_mangrove_canopy.py

```diff
@@ -249,3 +251,48 @@ def _extract(self) -> None:
         """Extract the dataset."""
         pathname = os.path.join(self.root, self.zipfile)
         extract_archive(pathname)
+
+    def plot(  # type: ignore[override]
+        self,
+        sample: Dict[str, Tensor],
+        show_titles: bool = True,
+        suptitle: Optional[str] = None,
+    ) -> plt.Figure:
+        """Plot a sample from the dataset.
+
+        Args:
+            sample: a sample returned by :meth:`RasterDataset.__getitem__`
+            show_titles: flag indicating whether to show titles above each panel
+            suptitle: optional string to use as a suptitle
+
+        Returns:
+            a matplotlib Figure with the rendered sample
+        """
```

Comment:
```suggestion
        
        .. versionadded:: 0.3
        """
```

Reply:
Is this necessary, because this entire dataset will only be added with version 0.3?

Reply:
Sorry for the confusion, this isn't necessary since this dataset has never been released, so as far as release code goes the plot method has always behaved this way. Can you remove this?